MobileWallet/Screens/Profile/ProfileViewController.swift

```diff
@@ -201,14 +148,10 @@ class ProfileViewController: UIViewController {
         }
     }
 
-    private func customizeSeparatorView() {
-        separatorView.backgroundColor = Theme.shared.colors.profileSeparatorView!
-    }
-
     private func customizeMiddleLabel() {
         let middleLabelText = String(
             format: NSLocalizedString(
-                "Transacting in person? Others can scan this QR code from the Tari Aurora App to send you %@.",
+                "This is your QR code for your friends to scan to quickly send you %@.",
```

Comment:
Is this a new copy change? We still have the old text on Android.

Reply:
just according to https://zpl.io/2pG9oJE

Reply:
@SerhiiShovkoplias This copy has been around for a long time, and there hasn't been any complaint from the product team. It was probably taken from the copy update document in Google Drive. Zeplin screens do not always have the latest copies, keeping this line as it was seems safer to me.